packages/ui-kit/src/lib/colors/index.ts

```diff
@@ -9,8 +9,10 @@ export const colors = {
   main_black: '#333333',
   warning_yellow: '#FF910F',
   problem_red: '#FF372A',
+  red_main: '#FF372A',
```

Comment:
It's exactly the same as the color above. Are they separated intentionally?

Reply:
Didn't notice that